docs/configuration.rst

```diff
@@ -0,0 +1,91 @@
+Configuration
+=============
+
+You can set the following options in your ``conf.py`` file.
+
+.. confval:: multiproject_projects
+
+   A dictionary defining the projects that will share the configuration.
+   You must define at least one project.
+
+   Example:
+
+   .. code-block:: python
+
+      multiproject_projects = {
+         "dev": {},
+         "user": {},
+      }
+
+   Each project can have the following options:
+
+   .. confval:: path
+
+      By default it's the same as the name of the project.
+      It can be a relative (to the conf.py file) or an absolute path.
```

Comment:
```suggestion
      Path to the nested project's ``conf.py`` file.
      By default, the key name used in ``multiproject_projects`` is also the directory name for the nested project's ``conf.py`` file.
      This can be a relative path (to the main ``conf.py`` file) or an absolute path.
```